packages/amplify-cli-core/src/state-manager/pathManager.ts

```diff
@@ -156,6 +161,18 @@ export class PathManager {
 
   getDeploymentSecrets = (): string => path.normalize(path.join(this.getDotAWSAmplifyDirPath(), PathConstants.DeploymentSecretsFileName));
 
+  getHooksDirPath = (projectPath?: string): string =>
```

Comment:
I don't think it makes sense for these hook-specific paths to be part of PathManager (especially not the sample file paths) or StateManager, below.

I would recommend creating a `hookStateManager` object that can expose functions for all hook state management related operations